lib/index-basic.js

```diff
@@ -16,6 +16,9 @@ Plotly.register([
 ]);
 
 // locales
-Plotly.register(require('./locales.js'));
+Plotly.register([
+    require('./locale-en.json'),
+    require('./locale-en-us.json')
```

Comment:
Ha. Webpack users won't like that. Webpack (v1 at least) needs a special loader to bundle JSON files unlike browserify. See https://github.com/plotly/plotly.js/pull/183

Reply:
ok, back to js https://github.com/plotly/plotly.js/pull/2195/commits/1275661c6c4d84cbbdf795f764623515b7c0d566